Dockerfile

```diff
@@ -10,8 +10,6 @@ COPY nginx.conf auth.htpasswd docker-entrypoint.sh ./
 
 USER root
 RUN chmod 777 /docker-entrypoint.sh
-RUN chmod -R 777 /usr/share/nginx/html/
-RUN echo "nginx on CentOS7" > /usr/share/nginx/html/index.html
```

Comment:
Why are you removing this? Not against removing it, but want to understand why.

Reply:
it's broken (it's a separate commit)